src/main/java/ml/shifu/shifu/core/Scorer.java

```diff
@@ -386,7 +397,29 @@ public MLData call() {
                         return ((GenericModel) model).compute(pair.getInput());
                     }
                 };
-                if (multiThread) {
+                if(multiThread) {
+                    tasks.add(callable);
+                } else {
+                    try {
+                        modelResults.add(callable.call());
+                    } catch (Exception e) {
+                        log.error("error in model evaluation", e);
+                    }
+                }
+            } else if(model instanceof WDLModel) {
+                final WDLModel wdl = (WDLModel) model;
+                // if(wdl.getInputCount() != pair.getInput().size()) {
+                // throw new RuntimeException("WDL and input size mismatch: wdl input Size = " + wdl.getInputCount()
+                // + "; data input Size = " + pair.getInput().size());
+                // }
```

Comment:
seems we can have exception here to quick identify input inconsistent issue

Reply:
I updated norm in eval for WDL model. For not selected columns, a "-1" will be inserted into data array to keep column num info.